<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalDayClass</class>
 <widget class="QWidget" name="CalDayClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>1050</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>1050</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CalDay</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
border:none;
background-color:rgb(247,247,247)
}</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>620</width>
     <height>660</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>620</width>
     <height>660</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>620</width>
     <height>660</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:15px ;color:rgb(0,0,0) </string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>622</width>
     <height>364</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px;background-color:rgb(255,133,21);color:rgb(255,255,255)</string>
      </property>
      <property name="text">
       <string>SwitchMode</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>开始日期</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QDateEdit" name="dateEdit">
      <property name="minimumSize">
       <size>
        <width>620</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>620</width>
        <height>110</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>31</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px ;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>结束日期</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="minimumSize">
       <size>
        <width>620</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>620</width>
        <height>110</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>31</pointsize>
        <italic>false</italic>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:15px ;</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>32</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>相差天数:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Calculator.qrc"/>
 </resources>
 <connections/>
</ui>
